sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/converters/CommunicationUserIdentifierConverter.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callautomation.implementation.converters;
+
+import com.azure.communication.callautomation.implementation.models.CommunicationUserIdentifierModel;
+import com.azure.communication.callautomation.implementation.models.PhoneNumberIdentifierModel;
+import com.azure.communication.common.CommunicationUserIdentifier;
+import com.azure.communication.common.PhoneNumberIdentifier;
+
+/**
+ * A converter for {@link PhoneNumberIdentifier} and {@link PhoneNumberIdentifierModel}
```

Comment:
is this not CommunicationUserIdentifier converter instead of PhoneNumberIdentifier converter?